src/test/java/com/sap/oss/phosphor/fosstars/tool/report/OssRulesOfPlayGitHubIssuesReporterTest.java

```diff
@@ -47,4 +63,59 @@ public void testPrintTitleAndBody() throws IOException {
     }
   }
 
+  @Test
+  public void testIssuesDisabled() throws IOException {
+    OssRulesOfPlayRating rating = RatingRepository.INSTANCE.rating(OssRulesOfPlayRating.class);
+    GitHubProject project = new GitHubProject("test", "project");
+    project.set(
+        new RatingValue(
+            new ScoreValue(rating.score()).set(Score.MIN).confidence(8.0).usedValues(
+                new BooleanValue(OssFeatures.HAS_LICENSE, false)),
+            OssRulesOfPlayRating.OssRulesOfPlayLabel.FAILED));
+
+    GitHubDataFetcher ghDataFetcher = mock(GitHubDataFetcher.class);
+
+    try {
+      OssRulesOfPlayGitHubIssuesReporter reporter = new OssRulesOfPlayGitHubIssuesReporter(
+          ghDataFetcher,
+          new OssRulesOfPlayRatingMarkdownFormatter(new OssRulesOfPlayAdvisor()));
+
+      when(ghDataFetcher.gitHubIssuesFor(any(), any())).thenReturn(Lists.newArrayList());
+      when(ghDataFetcher.createGitHubIssue(any(), any(), any())).thenThrow(
+          GHFileNotFoundException.class);
+
+      reporter.createIssuesFor(project);
+      verify(ghDataFetcher, times(1)).createGitHubIssue(any(), any(), any());
+    } finally {
+      FileUtils.forceDeleteOnExit(CONFIG_PATH.toFile());
```

Comment:
The test case doesn't seem to create this file.

Reply:
Indeed - missed to delete it after copy/pasting, will be corrected, thanks!